src/index.ts

```diff
@@ -83,6 +83,7 @@ const react = (_options?: Options): PluginOption[] => {
         esbuild: false,
         optimizeDeps: {
           include: [`${options.jsxImportSource}/jsx-dev-runtime`],
+          esbuildOptions: { jsx: "transform" },
```

Comment:
Probably I'm missing something, but why is it here `"transform"` and in the other plugin `"automatic"`? https://github.com/vitejs/vite-plugin-react/commit/96b66ed5e75cfc325a661e0d13b700cc275cfa60#diff-d1f942b9a74f586a80bca3733150b7c692a6460e8350bc19cbbad1d53f95b356R134

Reply:
I was just try things locally and didn't revert 🙈